executor/aggfuncs/aggfunc_test.go

```diff
@@ -234,6 +234,33 @@ func distinctUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType)
 	return memDeltas, nil
 }
 
+func noZeroUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas []int64, err error) {
+	for i := 0; i < srcChk.NumRows(); i++ {
+		row := srcChk.GetRow(i)
+		if i > 0 {
+			memDeltas = append(memDeltas, int64(0))
+			continue
+		}
+		switch dataType.Tp {
+		case mysql.TypeString:
+			val := row.GetString(0)
+			memDeltas = append(memDeltas, int64(len(val)))
+		case mysql.TypeJSON:
+			json := row.GetJSON(0)
```

Comment:
the `json` is a package name, we should rename this var.

Reply:
It's been changed.